### PR TITLE
added Invalid object UI

### PR DESCRIPTION
**Pull request by ryandillinfelton**
Adds a component for invalid objects. [Jira issue](
https://projects.engineering.redhat.com/browse/RHCLOUD-2101)
Invalid-Object-Component

### CODE REVIEW

**Comment by ryelo** [comment]
@ryandillinfelton Do you mind attaching a screenshot to this PR too? Just edit the original description.

**Comment by ryandillinfelton** [comment]
> @ryandillinfelton Do you mind attaching a screenshot to this PR too? Just edit the original description.

Added it

File: packages/components/src/Components/InvalidObject/icon-404.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (karelhala) [inline packages/components/src/Components/InvalidObject/icon-404.js]: Is this icon missing in PF4 icons?

Comment (ryandillinfelton) [reply packages/components/src/Components/InvalidObject/icon-404.js]: I believe so

File: packages/components/src/Components/InvalidObject/InvalidObject.test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import InvalidObject from './InvalidObject';
+
+describe('InvalidObject component', () => {
+    it('should render', () => {
+        const wrapper = shallow(<InvalidObject/>)
```
[the remark this answers is not in the document]
Comment (ryandillinfelton) [reply packages/components/src/Components/InvalidObject/InvalidObject.test.js]: Switched it to render. I had to add <Router> in for it to work

**Comment by ryelo** [comment]
@ryandillinfelton Once this is merged, can you open a PR in https://github.com/redhatinsights/insights-frontend-storybook?

**Comment by ryandillinfelton** [comment]
> @ryandillinfelton Once this is merged, can you open a PR in https://github.com/redhatinsights/insights-frontend-storybook?

Sure thing! I'll add some documentation for it when it's in

**Comment by karelhala** [comment]
Released in new version [@redhat-cloud-services/frontend-components/v/0.0.10](https://www.npmjs.com/package/@redhat-cloud-services/frontend-components/v/0.0.10)

**Pull request merged**